test/parallel/test-fs-fsync.js

```diff
@@ -1,32 +1,26 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var path = require('path');
-var fs = require('fs');
-var successes = 0;
+const path = require('path');
+const fs = require('fs');
 
-var file = path.join(common.fixturesDir, 'a.js');
+const file = path.join(common.fixturesDir, 'a.js');
 
-fs.open(file, 'a', 0o777, function(err, fd) {
-  if (err) throw err;
+const fdatasyncSync = common.mustCall(fs.fdatasyncSync);
+const fsyncSync = common.mustCall(fs.fsyncSync);
```

Comment:
@santigimeno Great question! I went back and forth with this before submitting. On one hand, I think having them makes the test a little more explicit and self documenting. On the other hand, they are not really needed since if either function fails, the test should throw an error. I am more than happy to remove them. Thoughts?

Thoughts?